src/commands/package.js

```diff
@@ -29,6 +29,11 @@ export default asyncCommand({
 			demandOption: true,
 			requiresArg: true
 		},
+		zimbraXVersion: {
+			description: 'Minimum required version of Zimbra X that the Zimlet is compatible with.',
```

Comment:
Given the wording here, I would assume the semver statement below would be `>=${zimbraXVersion}`

Reply:
If we accept semver here, I would state that and maybe put a link to semver.org.  Maybe default it to `^1.0.0` rather than having `demandOption: true`

Reply:
Good idea, I made it more clear that this is 1. semver and 2. a version range, not a single version.

Set default to `>=1.0.0`